pkg/util/sys/device.go

```diff
@@ -481,3 +482,17 @@ func parseUdevInfo(output string) map[string]string {
 	}
 	return result
 }
+
+// ListDevicesChild list all child available on a device
+func ListDevicesChild(executor exec.Executor, device string) ([]string, error) {
+	cmd := "lsblk for child"
+	var childList []string
+
+	childListRaw, err := executor.ExecuteCommandWithOutput(false, cmd, "lsblk", "--noheadings", "--pairs", path.Join("/dev", device))
+	if err != nil {
+		return childList, fmt.Errorf("failed to list child devices of %q. %+v", device, err)
```

Comment:
I know this is functionally equivalent, but at a glance, I think what I suggest makes it more clear that a failure is intended to return empty.

```suggestion
		return []string{}, fmt.Errorf("failed to list child devices of %q. %+v", device, err)
```